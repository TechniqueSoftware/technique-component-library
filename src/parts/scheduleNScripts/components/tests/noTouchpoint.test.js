import React from 'react'
import { render, screen } from '@testing-library/react'
import NoTouchPoint from '../../statusSchedules/noTouchpoint'

describe('Schedule and scripts <StatusSchedules />', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<NoTouchPoint />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('renders title on left', () => {
    render(<NoTouchPoint />)
    expect(screen.getByText('No touchpoints have been configured')).toBeInTheDocument()
  })
})
