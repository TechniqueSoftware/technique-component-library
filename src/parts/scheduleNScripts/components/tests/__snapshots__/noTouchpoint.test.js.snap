// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schedule and scripts <StatusSchedules /> should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="
  touchpoint
  empty
  
  rounded
  p-2
  overflow-hidden
  m-0
  justify-content-center
  align-items-center
  "
  >
    <p
      class="m-0 text-center fw-bold"
    >
      No touchpoints have been configured
    </p>
  </div>
</DocumentFragment>
`;
